### PR TITLE
Add changelog 6.1.0

### PR DESCRIPTION
**Pull request by will-moore**
changelog - see https://github.com/will-moore/figure/blob/changelog_6.1.0/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+- Zoom labels ([#527](https://github.com/ome/omero-figure/pull/527))
+- Add Cyan to colour choosers ([#516](https://github.com/ome/omero-figure/pull/516))
+- Align panels to right & bottom ([#520](https://github.com/ome/omero-figure/pull/520))
+- Right vertical labels ([#519](https://github.com/ome/omero-figure/pull/519))
+- Edit ID Z stack ([#518](https://github.com/ome/omero-figure/pull/518))
+- Manual Zoom level ([#517](https://github.com/ome/omero-figure/pull/517))
+
+Other changes:
+- Allow Pillow 10.x for figure export ([#523](https://github.com/ome/omero-figure/pull/523))
+- Improve developer instructions ([#515](https://github.com/ome/omero-figure/pull/515))
```
Comment (pwalczysko) [inline CHANGELOG.md:20]: Note: When testing the PR #515 , I encountered errors

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+- Labels from ALL Key-Value pairs ([#528](https://github.com/ome/omero-figure/pull/528))
+- Zoom labels ([#527](https://github.com/ome/omero-figure/pull/527))
+- Add Cyan to colour choosers ([#516](https://github.com/ome/omero-figure/pull/516))
+- Align panels to right & bottom ([#520](https://github.com/ome/omero-figure/pull/520))
+- Right vertical labels ([#519](https://github.com/ome/omero-figure/pull/519))
+- Edit ID Z stack ([#518](https://github.com/ome/omero-figure/pull/518))
+- Manual Zoom level ([#517](https://github.com/ome/omero-figure/pull/517))
+
+Other changes:
+- Allow Pillow 10.x for figure export ([#523](https://github.com/ome/omero-figure/pull/523))
```
Comment (pwalczysko) [inline CHANGELOG.md:19]:
```suggestion
- Do not use a function removed in Pillow 10.x ([#523](https://github.com/ome/omero-figure/pull/523))
```

Reading the title of the PR, this makes more sense to me.

**Review by pwalczysko — CHANGES_REQUESTED** [review]
2 issues, one formulation and one unmerged PR (potentially bigger one)

File: CHANGELOG.md (hunk 2 above)
Comment (will-moore) [reply CHANGELOG.md]: How about that change?

Comment (pwalczysko) [reply CHANGELOG.md]: yes, happy with https://github.com/ome/omero-figure/pull/534/commits/28c564f05a962e06578b48b74887d9196942151d, thank you

**Comment by pwalczysko** [comment]
Happy with https://github.com/ome/omero-figure/pull/534/commits/28c564f05a962e06578b48b74887d9196942151d - still the docker workflow to solve though https://github.com/ome/omero-figure/pull/534#discussion_r1425733722 - but rather on the other PR I guess

**Comment by will-moore** [comment]
Removed #515 from changelog (since we're not merging it till Docker sorted out).

OK to merge this PR and go ahead with release? @pwalczysko @jburel ?

**Comment by Rdornier** [comment]
@will-moore  - Thank you very much for the release !

> Sorry it took so long (various reasons)

No worries

**Pull request merged**